BUILDING.md

```diff
@@ -30,3 +30,66 @@ v1.1.0
+
+   a. **Import Project ->** select *build.gradle* file from Geode repository root and press **Open**.
+   b. Enable **Use auto-import**
+   c. Enable **Create separate module per source set**
+   d. Select **Use Project JDK 1.8.0_*nnn*** where *nnn* is latest build required for Geode
+   e. Expand **Global Gradle settings** and edit **Gradle VM options:** to contain `-DUSING_INTELLIJ=true` and press **OK**.
+
+   From an existing IntelliJ project (skip this if you imported from the **Welcome IntelliJ IDEA** window):
+
+   a. Navigate to **File -> New -> Project from Existing Sources...**, select *build.gradle* file from Geode repository root and press **Open**. Then follow b. through e. above.
+
+3. Change Code Style Scheme to GeodeStyle
+
+   Navigate to **IntelliJ IDEA -> Preferences... -> Editor -> Code Style**. Select *GeodeStyle* in Scheme drop-down box if it already exists.
+
+   To define the *GeodeStyle* in **Scheme**, select the gear icon next to the drop-down box, click **Import Scheme ->** and select **IntelliJ IDEA code style XML**. Select *etc/intellij-java-modified-google-style.xml* from Geode repository root, enter **To:** *GeodeStyle*, check **Current scheme** and press **OK**.
+
+4. Make Apache the default Copyright
+
+   Navigate to **IntelliJ IDEA -> Preferences... -> Editor -> Copyright**. Select *Apache* in drop-down box **Default project copyright**.
+
+   To define *Apache* in **Copyright**, navigate to **IntelliJ IDEA -> Preferences... -> Editor -> Copyright -> Copyright Profiles**. Click **+** to add a new project. Enter *Apache* as the **Name** and enter the following block without asterisks or leading spaces:
+
+   ```text
+   Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+   agreements. See the NOTICE file distributed with this work for additional information regarding
+   copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance with the License. You may obtain a
+   copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software distributed under the License
+   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+   or implied. See the License for the specific language governing permissions and limitations under
+   the License.
+   ```
+
+   ...then return to **Copyright** and select *Apache* in drop-down box **Default project copyright**.
+
+   Navigate to **IntelliJ IDEA -> Preferences... -> Editor -> Copyright -> Formatting**. Uncheck **Add blank line after** and press **OK**.
+
+5. Rebuild Project
+
+   Navigate to **Build -> Rebuild Project** and the full project should compile without errors.
+
+   Some optional sanity tests to make sure things are working properly:
+
+   a) Try looking up classes in both closed and open with Command-O.
```

Comment:
You should remove any mention of "closed"

Reply:
Missed that one, will clean it up